lib/src/flutter/runners/gherkin_integration_test_runner.dart

```diff
@@ -254,12 +271,8 @@ abstract class GherkinIntegrationTestRunner {
   }
 
   @protected
-  Future<StepResult> runStep(
-    String step,
-    Iterable<String> multiLineStrings,
-    dynamic table,
-    TestDependencies dependencies,
-  ) async {
+  Future<StepResult> runStep(String step, Iterable<String> multiLineStrings,
+      dynamic table, TestDependencies dependencies, bool hasToSkip) async {
```

Comment:
Please add a comma after `hasToSkip` we should probably get the latests flutter_lints packages sorted in library